modules/announcements/src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
Is the CSS necessary, does it not import the CSS from the parent app ? 

cc @arajput

Reply:
this is default file by CRA. it is used only when module runs as stand alone.